packages/cli/templates/webcomponents/igc-ts/projects/empty/files/index.html

```diff
@@ -15,7 +15,7 @@
     :host {
       display: block;
       padding: 25px;
-      color: var(--test-component-test-text-color, #000);
+      color: hsla(var(--test-component-test-text-color, #000));
```

Comment:
`--test-component-test-text-color` not form this PR but this should be removed.
Also this will break now.
- It was `var(--test-component-test-text-color, #000)` which will retrun `#000`.
- Now calling `hsla(var(--test-component-test-text-color, #000))` will call `hsla(#000)` which is not valid.